web/client/codechecker_client/cmd_line_client.py

```diff
@@ -1643,3 +1643,63 @@ def handle_list_run_histories(args):
                          h.description if h.description else ''))
 
         print(twodim.to_str(args.output_format, header, rows))
+
+
+def handle_export(args):
+
+    stream = 'stderr'
+    init_logger(args.verbose if 'verbose' in args else None, stream)
+
+    client = setup_client(args.product_url)
+    run_filter = process_run_filter_conditions(args)
+
+    runs = get_run_data(client, run_filter)
+    if not runs:
+        LOG.warning("No runs found")
+        sys.exit(1)
+
+    report = client.exportData(run_filter)
+    print(CmdLineOutputEncoder().encode(report))
+
+
+def handle_import(args):
+    """
+    Argument handler for the 'CodeChecker cmd import' subcommand
+    """
+
+    init_logger(args.verbose if 'verbose' in args else None)
+
+    client = setup_client(args.product_url)
+
+    data = util.load_json_or_empty(args.input, default=None)
+    if not data:
+        LOG.error("Failed to import data!")
+        sys.exit(1)
+
+    # Convert Json comments into CommentDataList
+    comment_data_list = defaultdict(list)
+    for bug_hash, comments in data['comments'].items():
+        for comment in comments:
+            comment_data = ttypes.CommentData(
+                id=comment['id'],
+                author=comment['author'],
+                message=comment['message'],
+                createdAt=comment['createdAt'],
+                kind=comment['kind'])
+            comment_data_list[bug_hash].append(comment_data)
+
+    # Convert Json reviews into ReviewData
+    review_data_list = {}
+    for bug_hash, review in data['reviewData'].items():
+        review_data_list[bug_hash] = ttypes.ReviewData(
+            status=review['status'],
+            comment=review['comment'],
+            author=review['author'],
+            date=review['date'])
+
+    exportData = ttypes.ExportData(comments=comment_data_list,
+                                   reviewData=review_data_list)
+
+    status = client.importData(exportData)
+    if status:
+        LOG.info("Import successful!")
```

Comment:
We need to handle the use case when the import is failed:
```suggestion
    if not status:
        LOG.error("Failed to import data!")
        sys.exit(1)

    LOG.info("Import successful!")
    
```